tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -646,7 +646,7 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
     # Command line tool specific params
     common_group.add_argument('--output_model',
                               help='This parameter is used to name output .xml/.bin files with converted model.')
-    common_group.add_argument('--compress_to_fp16', action='store_true',
+    common_group.add_argument('--compress_to_fp16', action='store_true', default=True,
```

Comment:
Keep this parameter in `mo` CLI tool only. Not propagate to Python API `convert_model`.

Reply:
It is kept in cli tool only.  If user will try to do ovc.convert(..., compress_to_fp16=True/False) he/she will end up having unexpected keyword argument error